src/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { Suspense } from 'react';
+import { BrowserRouter } from 'react-router-dom';
+
+import { createRoot } from 'react-dom/client';
+
+import App from './App';
+import configureI18n from './i18n';
+import reportWebVitals from './reportWebVitals';
+
+configureI18n();
+
+const container = document.getElementById('root') as HTMLElement;
+const root = createRoot(container);
+root.render(
+  <React.StrictMode>
+    <Suspense fallback="loading">
+      <BrowserRouter>
+        <App />
+      </BrowserRouter>
+    </Suspense>
+  </React.StrictMode>
+);
+
+// If you want to start measuring performance in your app, pass a function
+// to log results (for example: reportWebVitals(console.log))
+// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
+reportWebVitals();
```

Comment:
I don't think we need this. What do you think about getting rid of it 🤔

Reply:
@nvminhtue removed in [b68ff78](https://github.com/manh-t/react-survey/pull/39/commits/b68ff7877896343f127df50578d91e3d4360f9f1)